adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1542,7 +1542,8 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
    */
   def loadIndexedBam(
     pathName: String,
-    viewRegions: Iterable[ReferenceRegion])(implicit s: DummyImplicit): AlignmentRecordRDD = LoadIndexedBam.time {
+    viewRegions: Iterable[ReferenceRegion],
+    stringency: ValidationStringency = ValidationStringency.STRICT)(implicit s: DummyImplicit): AlignmentRecordRDD = LoadIndexedBam.time {
```

Comment:
Can you add scaladoc for this?

Reply:
Also, can you add this to the other definition of `loadIndexedBam`?

Reply:
I added a scaladoc. We can't add this to the loadIndexed bam due to the same overloadding issue we had with loadIndexedVcf

Reply:
Ah, gotcha! SGTM.